tools/explorer/pkg/escrow/escrow.go

```diff
@@ -170,63 +169,89 @@ func (e *Escrow) checkReservations() error {
 		return errors.Wrap(err, "failed to load active reservations from escrow")
 	}
 
-	var slog zerolog.Logger
-
 	for _, escrowInfo := range reservationEscrows {
-		slog = log.With().
-			Str("address", escrowInfo.Address).
-			Int64("reservation_id", int64(escrowInfo.ReservationID)).
-			Logger()
-
-		requiredValue := xdr.Int64(0)
-		// calculate total amount needed for reservation
-		for _, escrowAccount := range escrowInfo.Infos {
-			requiredValue += escrowAccount.TotalAmount
-		}
-
-		balance, _, err := e.wallet.GetBalance(escrowInfo.Address, escrowInfo.ReservationID)
-		if err != nil {
-			slog.Error().Err(err).Msgf("failed to verify escrow account balance")
+		if err := e.checkReservationPaid(escrowInfo); err != nil {
+			log.Error().
+				Str("address", escrowInfo.Address).
+				Int64("reservation_id", int64(escrowInfo.ReservationID)).
+				Err(err)
```

Comment:
This line won't print anything zerolog requires to have `Msg`, `Msgf` or just `Send` if you don't want to add a message.